docker-compose.yml

```diff
@@ -216,3 +216,5 @@ services:
     logging: *default-logging
     labels:
       - "logging=true"
+  lmb-proxy:
+    image: lblod/lmb-sparql-proxy:0.0.1
```

Comment:
you can use a github reference for these, it's pretty neat
e.g. this pr https://github.com/lblod/app-reglementaire-bijlage/pull/77